reason-react-native/src/components/FlatList.re

```diff
@@ -178,49 +182,3 @@ external make:
   ) =>
   React.element =
   "FlatList";
-
-// VirtualizedList methods
```

Comment:
Don't need these here anymore, as they are already defined on `VirtualizedList.element` and we can convert using `asVirtualizedList`.